controllers/schools.js

```diff
@@ -148,10 +157,9 @@ const getDeleteHandler = (service) => {
 const getHandler = async (req, res) => {
   const itemsPerPage = req.query.limit || 10;
   const currentPage = parseInt(req.query.p) || 1;
-
-  const [federalStates, data, storageProvider] = await Promise.all([
-    api(req).get('/federalStates'),
-    api(req).get('/schools', {
+  try {
+    const federalStates = await api(req).get('/federalStates');
```

Comment:
Why do you removed the Promise.all?

Reply:
fixed (I forgot it there for easier debugging)